packages/component/src/Activity/Bubble.js

```diff
@@ -18,22 +20,71 @@ const ROOT_CSS = css({
   }
 });
 
-const Bubble = ({ 'aria-hidden': ariaHidden, children, className, fromUser, nub, styleSet }) => (
-  <div
-    aria-hidden={ariaHidden}
-    className={classNames(
-      ROOT_CSS + '',
-      styleSet.bubble + '',
-      { 'from-user': fromUser, webchat__bubble_has_nub: nub },
-      className + '' || ''
-    )}
-  >
-    <div className="webchat__bubble__content">{children}</div>
-    {nub && !!(fromUser ? styleSet.options.bubbleFromUserNubSize : styleSet.options.bubbleNubSize) && (
-      <div className="webchat__bubble__nub" />
-    )}
-  </div>
-);
+function acuteNubSVG(nubSize, strokeWidth, side, upSideDown = false) {
+  const halfNubSize = nubSize / 2;
+  const halfStrokeWidth = strokeWidth / 2;
+  const horizontalTransform =
+    side === 'bot' ? '' : `translate(${halfNubSize} 0) scale(-1 1) translate(${-halfNubSize} 0)`;
```

Comment:
comments to explain this?

Reply:
Added comments about the horizontal and vertical mirroring.